{"name": "Cоздание пользователя, который уже зарегистрирован", "status": "passed", "description": "Проверка: статус 403, сообщение об ошибке, что такой уже существует.", "start": 1724955235304, "stop": 1724955235691, "uuid": "8b8cbe79-344c-48d5-9f44-ff6e3a575082", "historyId": "91c577d262adcb3c4bed96306c3f4a57", "testCaseId": "91c577d262adcb3c4bed96306c3f4a57", "fullName": "tests.test_register_user.TestRegisterUser#test_create_existing_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_register_user"}, {"name": "subSuite", "value": "TestRegisterUser"}, {"name": "host", "value": "LAPTOP-74V5TFPG"}, {"name": "thread", "value": "8416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_register_user"}]}